src/ucp/core/ucp_worker.c

```diff
@@ -231,8 +235,12 @@ static void ucp_worker_remove_am_handlers(ucp_worker_h worker)
                                         UCT_IFACE_FLAG_AM_ZCOPY))) {
             continue;
         }
-        for (am_id = 0; am_id < UCP_AM_ID_LAST; ++am_id) {
-            if (context->config.features & ucp_am_handlers[am_id].features) {
+        for (am_id = 1; am_id < UCP_AM_ID_LAST; ++am_id) {
```

Comment:
shall we define UCP_AM_ID_FIRST?

Reply:
> shall we define UCP_AM_ID_FIRST?

already defined to send multi-fragment message. used UCP_AM_ID_WIREUP instead

Reply:
oh..
maybe rename UCP_AM_ID_FIRST to UCP_AM_ID_AM_FIRST (also SINGLE and MIDDLE)
@brminich WDYT?

Reply:
> oh..
> maybe rename UCP_AM_ID_FIRST to UCP_AM_ID_AM_FIRST (also SINGLE and MIDDLE)
> @brminich WDYT?

renamed